redfish-core/lib/log_services.hpp

```diff
@@ -1098,12 +1098,19 @@ inline void createDump(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
 
             dumpPath = "/xyz/openbmc_project/dump/resource";
 
-            createDumpParams.emplace_back(
-                std::make_pair("com.ibm.Dump.Create.CreateParameters.VSPString",
-                               resourceDumpParams[1]));
-            createDumpParams.emplace_back(
-                std::make_pair("com.ibm.Dump.Create.CreateParameters.Password",
-                               resourceDumpParams[2]));
+            if (resourceDumpParams.size() >= 2)
```

Comment:
You shouldn't throw an error if the size is less than two? or is there valid resource dumps where we don't set this?

Reply:
yes, I think there can be.

Reply:
there is a check added above for size >3